tests/unit/test-util/test.adb.js

```diff
@@ -223,12 +223,28 @@ describe('utils/adb', () => {
         testFn: (adbUtils) => adbUtils.discoverInstalledFirefoxAPKs('device1'),
       });
 
+      sinon.assert.calledOnce(adb.fakeADBDevice.shell);
```

Comment:
Restore the test coverage that verified the received parameters (`calledWith`).

Reply:
This is intentionally removed, assuming this test case purpose is to check recovery from the adb failure.
It should not depend on the command really. The command is now tested in the positive case below.

Reply:
This is a negative test and it's named 'rejects an UsageError on adb binary not found'.
With the change the first shell command is no longer `pm list packages` but `am get-current-user`.
I'm quite convinced the command is not relevant in this test as the test purpose is clearly to test a recovery from failure on accessing the adb shell. Both commands are now being verified in other tests. Might be a matter of taste but verifying the same behavior in multiple tests may cause unnecessary coupling and make the code harder ro change.
Not sure would one want to change this specific test and other tests when another command is issued as first one in this flow.